src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -74,6 +156,62 @@ public static String prettyPrintSequenceRecords ( SAMSequenceDictionary sequence
         RIGHT_TAIL
     }
 
+    public static final int SAM_READ_PAIRED_FLAG = 0x1;
```

Comment:
htsjdk does have declarations of these, but they are private, so I had to copy them unfortunately.